scripts/plotting/polar_map.py

```diff
@@ -157,7 +162,7 @@ def polar_map(adfobj):
 
                 #Check if plot output directory exists, and if not, then create it:
                 if not plot_loc.is_dir():
-                    print("    {} not found, making new directory".format(plot_loc))
+                    print("\t    {} not found, making new directory".format(plot_loc))
```

Comment:
I know you didn't write it this way, but can you convert this to an f-string:

```suggestion
                    print(f"\t    {plot_loc} not found, making new directory")
```